packages/trader/src/Modules/Trading/Components/Elements/mobile-widget.tsx

```diff
@@ -0,0 +1,120 @@
+import React from 'react';
+import { Money } from '@deriv/components';
+import { Localize } from '@deriv/translations';
+import { getExpiryType, getDurationMinMaxValues, getLocalizedBasis } from '@deriv/shared';
+import { MultiplierAmountWidget } from 'Modules/Trading/Components/Form/TradeParams/Multiplier/widgets.jsx';
+import TradeParamsModal from '../../Containers/trade-params-mobile';
+import { observer, useStore } from '@deriv/stores';
+import { useTraderStore } from 'Stores/useTraderStores';
+
+const MobileWidget = observer(
+    ({ toggleDigitsWidget, is_collapsed }: { toggleDigitsWidget: () => void; is_collapsed: boolean }) => {
+        const { ui } = useStore();
+        const { onChangeUiStore } = ui;
+        const trade_store = useTraderStore();
+        const {
+            amount,
+            basis,
+            currency,
+            duration,
+            duration_min_max,
+            duration_unit,
+            form_components,
+            is_multiplier,
+            last_digit,
+            onChange,
+        } = trade_store;
+
+        const [is_open, setIsOpen] = React.useState(false);
+
+        React.useEffect(() => {
+            assertDurationIsWithinBoundary();
+        });
+
+        const assertDurationIsWithinBoundary = () => {
+            const contract_expiry_type = getExpiryType(trade_store);
+            const [min_value, max_value] = getDurationMinMaxValues(
+                duration_min_max,
+                contract_expiry_type,
+                duration_unit
+            );
+
+            if (contract_expiry_type === 'tick' && duration < Number(min_value)) {
+                onChangeUiStore({ name: `duration_${duration_unit}`, value: min_value });
+                onChange({ target: { name: 'duration', value: min_value } });
+            }
+            if (!(duration < Number(min_value)) && duration > Number(max_value)) {
+                onChangeUiStore({ name: `duration_${duration_unit}`, value: max_value });
+                onChange({ target: { name: 'duration', value: max_value } });
+            }
+        };
+
+        const toggleWidget = () => setIsOpen(!is_open);
+
+        const getHumanReadableDuration = () => {
+            if (!duration_unit) return '';
+            const lookup = {
+                t: [<Localize i18n_default_text='tick' key='t' />, <Localize i18n_default_text='ticks' key='ts' />],
```

Comment:
After I swapped `localize()` and `<Localize />` there was need to add keys to the array members in order to avoid TS error 
<img width="911" alt="Screenshot 2023-09-13 at 9 20 11 AM" src="https://github.com/kate-deriv/deriv-app/assets/121025168/72511460-ffc4-4ccb-9412-dbda8858db45">